android/src/main/AndroidManifest.xml

```diff
@@ -1,5 +1,3 @@
-
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-          package="com.snowplowanalytics.react.tracker">
-
+          package="com.snowplow.reactnativetracker">
```

Comment:
It is not needed, but it was caused by recreating the project with RN defaults – based on the project name, the generator went for this. It doesn't really matter because it's internal within the package, does not have any effect outside of tracker internals.